nixos/maintainers/scripts/gce/create-gce.sh

```diff
@@ -1,15 +1,27 @@
-#! /bin/sh -e
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p google-cloud-sdk
 
-BUCKET_NAME=${BUCKET_NAME:-nixos-images}
-export NIX_PATH=nixpkgs=../../../..
-export NIXOS_CONFIG=$(dirname $(readlink -f $0))/../../../modules/virtualisation/google-compute-image.nix
-export TIMESTAMP=$(date +%Y%m%d%H%M)
+set -euo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+NIXPKGS_ROOT="$DIR/../../../../"
```

Comment:
I think it's better to simply use `nixpkgs` from $NIX_PATH, similar to `nixos/maintainers/scripts/ec2/create-amis.sh`. I often need to tweak `create-amis.sh` temporarily (e.g. upload to one region first) but I do want to ensure that the Nixpkgs used to build the image corresponds cleanly to a channel, rather than my working directory. So I do `NIX_PATH=nixpkgs=channel:nixos-17.03 ./nixos/maintainers/scripts/ec2/create-amis.sh`.